docs/trulens_eval/logging.ipynb

```diff
@@ -130,6 +130,29 @@
     "tru.add_record(record)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "211d94f6",
+   "metadata": {},
+   "source": [
+    "### Log App Feedback\n",
+    "Capturing app feedback such as user feedback of the responses can be added with one call."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9a1393f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "thumb_result = True\n",
+    "tru.add_feedback(name=\"thumbs up result\", \n",
+    "                  record_id=record.record_id,\n",
+    "                  chain_id=truchain.chain_id, \n",
```

Comment:
Non-blocking: Ideally, we don't need to add a chain_id because record_ids are uuids. Is there a way to do that?

Reply:
Added it as a follow up. Probably can be a wider upgrade outside of this function too